docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -689,6 +689,36 @@ export default function PlainCssPriority() {
 
 **Note:** If you are using styled-components and have `StyleSheetManager` with a custom `target`, make sure that the target is the first element in the HTML `<head>`. If you are curious to see how it can be done, you can take a look at the [`StyledEngineProvider`](https://github.com/mui/material-ui/blob/master/packages/mui-styled-engine-sc/src/StyledEngineProvider/StyledEngineProvider.js) implementation in the `@mui/styled-engine-sc` package.
 
+5. Change the target container for `Portal`-related elements so that they are injected under the main app wrapper that was used in point 3. for setting up the `important` option in the Tailwind's config.
```

Comment:
```suggestion
5. Change the target container for `Portal`-related elements so that they are injected under the main app wrapper that was used in step 3 for setting up the `important` option in the Tailwind config.
```